share/instruments/instruments.xml

```diff
@@ -7734,6 +7734,33 @@
                   <genre>concertband</genre>
                   <genre>marching</genre>
             </Instrument>
+            <Instrument id="taiko">
+                  <family>drums</family>
+                  <trackName>Taiko</trackName>
+                  <longName>Taiko</longName>
+                  <shortName>Taiko</shortName>
```

Comment:
not really a short name, is it?

Reply:
I think it is fine.  It's not the only instrument with the same short and long name.
I've seen "Taiko Drum" for long name and "Taiko" for short.  But I'd rather not be forced to choose between single and plural.
Something like Tko. or T. feels a little wrong to me for some totally subjective reason, but I'm open to feedback.